run

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+let cmd = require('./build/lib/cmd')
+let fss = require('fs')
+let fs  = require('fs').promises
+
+let args = process.argv.slice(2)
+
+let no_validation = '--no-validation'
+let no_nvm_check  = '--no-nvm-check'
+let nvm_error = `
+You are not using Node Version Manager. Building the project will probably fail.
+Follow the link to learn how to use NVM: https://github.com/nvm-sh/nvm
+Please use '${no_nvm_check}' to skip this check (not recommended).
+`
+
+async function init () {
+    let initialized = fss.existsSync('.initialized')
+    if (!initialized) {
+        if(!args.includes(no_validation)) {
+            cmd.section('Validation')
+            if (!args.includes(no_nvm_check)) {
+                if (process.env.NVM_DIR == undefined) {
+                    throw nvm_error
+                }
+                // TODO[WD]: Why its not working?
+                // await cmd.check_version('nvm','0.35.2')
+            }
+            await cmd.check_version('npm','6.14.1')
+            await cmd.check_version('node','v12.16.1')
```

Comment:
I don't think that current CI script ensures that we get such version. If so, any external update to actions/images will break us. 
Either CI script should be strengthened or this checks weakened.